frontend/Dockerfile

```diff
@@ -2,7 +2,7 @@
 ARG VOLTO_VERSION
 FROM plone/frontend-builder:${VOLTO_VERSION} AS builder
 
-COPY --chown=node packages/core /app/packages/core
+COPY --chown=node packages/kitconcept-core /app/packages/core
```

Comment:
It should also be copied to /app/packages/kitconcept-core inside the container, no?